pkg/docs/docs.go

```diff
@@ -793,7 +793,7 @@ const docTemplate = `{
 var SwaggerInfo = &swag.Spec{
 	Version:          "v0.0.8",
 	Host:             "localhost:8080",
-	BasePath:         "/v1",
+	BasePath:         "/api/v1",
```

Comment:
Is this change auto-generated?

Reply:
no. I manually fixed this.

Reply:
Then the change will be overridden. Please run make swag to generate it.

Reply:
I already ran make swag. this file doesn't belong to swagger

Reply:
I already ran that. Probably it's generated... not sure